### ISSUE TITLE
Implement smiles predictor demo

### ISSUE THREAD

**Issue by zuzg** [zuzg/drug-screening#135]

**Comment by zuzg** [comment]
Low effort option ([DeepPurpose](https://github.com/mims-harvard/TDC/blob/master/tutorials/TDC_104_ML_Model_DeepPurpose.ipynb)): seems badly maintain, method `predict` requires dataframe with `Label` [🥴](https://emojipedia.org/woozy-face)

What is interesting, it gives the same predictions regardless of labels...

![image](https://github.com/zuzg/drug-screening/assets/81712088/cb87c367-2cf2-4717-8597-fbe71d7c6790)

In summary: not reliable, will try with ADMET
